custom/inddex/food.py

```diff
@@ -1,175 +1,188 @@
 from .fixtures import FixtureAccessor
 
-ORDERED_INDICATORS = [
-    'unique_respondent_id',
-    'location_id',
-    'respondent_id',
-    'recall_case_id',
-    'opened_date',
-    'opened_by_username',
-    'owner_name',
-    'recalled_date',
-    'recall_status',
-    'gender',
-    'age_years_calculated',
-    'age_months_calculated',
-    'age_range',
-    'pregnant',
-    'breastfeeding',
-    'supplements',
-    'urban_rural',
-    'food_code',
-    'food_name',
-    'recipe_name',
-    'caseid',
-    'reference_food_code',
-    'base_term_food_code',
-    'food_type',
-    'include_in_analysis',
-    'food_status',
-    'eating_time',
-    'time_block',
-    'fao_who_gift_food_group_code',
-    'fao_who_gift_food_group_description',
-    'user_food_group',
-    'already_reported_food',
-    'already_reported_food_case_id',
-    'already_reported_recipe',
-    'already_reported_recipe_case_id',
-    'already_reported_recipe_name',
-    'is_ingredient',
-    'recipe_case_id',
-    'ingr_recipe_code',
-    'ingr_fraction',
-    'ingr_recipe_total_grams_consumed',
-    'short_name',
-    'food_base_term',
-    'tag_1',
-    'other_tag_1',
-    'tag_2',
-    'other_tag_2',
-    'tag_3',
-    'other_tag_3',
-    'tag_4',
-    'other_tag_4',
-    'tag_5',
-    'other_tag_5',
-    'tag_6',
-    'other_tag_6',
-    'tag_7',
-    'other_tag_7',
-    'tag_8',
-    'other_tag_8',
-    'tag_9',
-    'other_tag_9',
-    'tag_10',
-    'other_tag_10',
-    'conv_method_code',
-    'conv_method_desc',
-    'conv_option_code',
-    'conv_option_desc',
-    'measurement_amount',
-    'conv_units',
-    'portions',
-    'nsr_conv_method_code_post_cooking',
-    'nsr_conv_method_desc_post_cooking',
-    'nsr_conv_option_code_post_cooking',
-    'nsr_conv_option_desc_post_cooking',
-    'nsr_measurement_amount_post_cooking',
-    'nsr_consumed_cooked_fraction',
-    'recipe_num_ingredients',
-    'conv_factor_food_code',
-    'conv_factor_base_term_food_code',
-    'conv_factor_used',
-    'conv_factor',
-    'fct_food_code_exists',
-    'fct_base_term_food_code_exists',
-    'fct_reference_food_code_exists',
-    'fct_data_used',
-    'fct_code',
-    'total_grams',
-    'energy_kcal_per_100g',
-    'energy_kcal',
-    'water_G_per_100g',
-    'water_g',
-    'protein_g_per_100g',
-    'protein_g',
-    'conv_factor_gap_code',
-    'conv_factor_gap_desc',
-    'fct_gap_code',
-    'fct_gap_desc',
+MISSING = ''
+
+IN_UCR = 'in_ucr'
+IS_RECALL_META = 'is_recall_meta'
```

Comment:
My aim here is to create a series of tags to document a bit about how each indicator is used and where it comes from (and to hopefully avoid a series of lists like "indicators, indicators_in_ucr, recall_meta_indicators, indicators_in_food_fixture...")